### PR TITLE
added module file for SAB2

### PR DESCRIPTION
**Pull request by cajt**
module.lb was missing, added one derived from SAB.

### CODE REVIEW

**Comment by salkinium** [comment]
I didn't add this on purpose, since there wasn't an example to compile and verify its functionality. Do you want to add one? (I assume you're using this in another project).

**Comment by cajt** [comment]
Well, of course, there is another project using it. Do you need an compile-able code example or a some type of test?

**Comment by salkinium** [comment]
> Do you need an compile-able code example or a some type of test?

Example I think is enough for now, unit test would be excellent. But SABx is not used by the RCA anymore, hence low maintenance from my side.

File: examples/stm32f4_discovery/sab2/main.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+modm::sab2::Interface<Usart1, 20> sabA;
+modm::sab2::Interface<Uart4, 20> sabB;
+
+int
+main()
+{
+	Board::initialize();
+	Board::LedBlue::set();
+
+	Usart1::connect<GpioB6::Tx>();
```
Comment (rleh) [inline examples/stm32f4_discovery/sab2/main.cpp]: You ~can~ should connect Tx and Rx in a single line:
```cpp
Usart1::connect<GpioB6::Tx, GpioB7::Rx>(); 
```

Comment (salkinium) [reply examples/stm32f4_discovery/sab2/main.cpp]: Not just *can*, you *should*, so that the API can check for pin group remap violations.

Comment (cajt) [reply examples/stm32f4_discovery/sab2/main.cpp]: I don't know if this is a request or general commentary. I interpreted it as a request and there is the fix.

**Review by salkinium — DISMISSED** [review]
Thanks!

**Pull request merged**